libraries/passive_sensitive.rb

```diff
@@ -0,0 +1,16 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: passive_sensitive.rb
+
+require 'chef/resource'
+
+# rubocop:disable Documentation
+class Chef
+  # Making the sensitive attribute passive for older chef versions
+  class Resource
+    def sensitive(_ = nil)
+    end
+  end
+end unless Chef::Resource.method_defined? :sensitive
+# rubocop:enable Documentation
```

Comment:
Instead of making this passive, what would be the work to make it just perform the same as later versions of chef?  Would it just be: https://github.com/chef/chef/blob/647e90ab9b778e92777f54c7201f70656afcf6cc/lib/chef/provider/execute.rb#L89-L91 ?